internal/sandbox/sandbox.go

```diff
@@ -380,6 +397,9 @@ func (s *podmanSandbox) execContainerCmd(execArgs []string) *exec.Cmd {
 		"exec",
 		s.container,
 	}
+	if s.command != "" {
+		args = append(args, s.command)
+	}
```

Comment:
I'm not sure the `Command` behaviour is the right approach here, and probably could be removed.

I think the behaviour of `Run` and `execContainerCmd` are confused be the presence of `Command`.

If `Command` is set, then `args[0]` is the first argument to `Command`.

But if `Command` is not set, then `args[0]` is the command that is executed.

Additionally, if `Run()` is called multiple times when `Command` is set, then the behaviour feels non-intuitive.

Reply:
Hm, I'll have to think more about this. Perhaps it should be possible to set and unset the command, and `Run` should be required to have a command set before it will work. I agree that "if `Command` is not set, then `args[0]` is the command that is executed" isn't great behaviour, and to be honest I only left it like that because that's what the behaviour was previously.

One issue is that dynamic analysis internally needs to reuse the same command with different arguments for each phase. But a custom command string comes from the command-line main file which doesn't know about this.

Reply:
I've resolved this by adding an extra argument to `Run` to pass the `command` on every call